react-native-libraries.json

```
@@ -12036,5 +12036,10 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/vokhuyetOz/react-native-draw-overlay",
```

```suggestion
    "githubUrl": "https://github.com/vokhuyetOz/react-native-draw-overlay",
    "npmPkg": "@vokhuyet/react-native-draw-overlay",
```

Hello. @vokhuyetOz, thanks for adding new library! 👍 

For the scoped packages supplying `npmPkg` manually is needed:
* https://github.com/react-native-community/directory?tab=readme-ov-file#npmpkg

Besides that, new entry looks good!